modular_skyrat/code/modules/uplink/uplink_clothing.dm

```diff
@@ -0,0 +1,11 @@
+/datum/uplink_item/suits/stealthsuit
+	name = "Tactical Stealth Suit"
+	desc = "A tight-fitting stealth suit, able to make you near completely invisible."
+	item = /obj/item/clothing/under/syndicate/stealthsuit
+	cost = 5
```

Comment:
Is this supposed to be a part of this?

Reply:
Ah, I see you have it a part of the code section

Reply:
I moved it to clothing because it makes more sense.